tests/integration/helpers/cluster.py

```diff
@@ -417,12 +417,66 @@ def __init__(
         self.mongo_port = get_free_port()
         self.mongo_no_cred_host = "mongo2"
         self.mongo_no_cred_port = get_free_port()
+        self.mongo_docker_port = "27017"
```

Comment:
But , for initializing  function i have to create separate variable with port number , previously it was hardcoded in same file , so i put it seperatly into cluster.py file , and for env_variables it will initialized after function call in cluster.py but we are not calling that function.

Reply:
As I understand they represent the same port, so it it logical that it should use the same origin. Otherwise it will break if you change one of them.

Reply:
i have not added anything from my side , i just separated the hardcoded things to another file.